File: game/Assets/Scripts/Network/Messages/UserInfo.cs
using System;

namespace Network
{
    public enum UserType
    {
        NONE = 0,
        MTURK,
        OPEN,
        GOOGLE,
        BOT
    }

    [Serializable]
    public class UserInfo
    {
        public string user_name;
        public UserType user_type;
    }
}

File: server/messages/user_info.py
from dataclasses import dataclass
from enum import IntEnum

from mashumaro.mixins.json import DataClassJSONMixin


class UserType(IntEnum):
    NONE = 0
    MTURK = 1
    OPEN = 2
    GOOGLE = 3
    BOT = 4


@dataclass(frozen=True)
class UserInfo(DataClassJSONMixin):
    user_name: str = ""
    user_type: UserType = UserType.NONE
